docs/discord.rst

```diff
@@ -35,7 +35,7 @@ Creating a Bot account is a pretty straightforward process.
     .. image:: /images/discord_bot_user_options.png
         :alt: How the Bot User options should look like for most people.
 
-7. Copy the token using the "Copy" button.
+7. Generate the token using the "Reset Token" button. You might have to confirm and enter your current 2FA Code.
```

Comment:
```suggestion
7. Generate the token using the "Reset Token" button. You have to confirm and enter your current 2FA Code, even if you don't have 2FA enabled.
```

Reply:
Is 2FA actually required and also promoted even if not enabled?

Reply:
wait nvm im wrongc